Dockerfile

```diff
@@ -20,11 +20,17 @@
 # docker run --net=host <image> -host-label=<label>
 ##
 
-FROM golang:1.4
+FROM golang:1.5.1
 MAINTAINER Madhav Puri <mapuri@cisco.com> (@mapuri)
 
+
+ENV http_proxy "http://proxy.esl.cisco.com:8080"
```

Comment:
I'm not sure if we need to change this file. 
Can you elaborate what was the intent of these changes?